setup.py

```diff
@@ -0,0 +1,8 @@
+"""
+Simple setuptools shim to allow editable installs from setup.cfg
+"""
```

Comment:
Maybe add the copyright and license header here, even if it is a trivial file, just for consistency.

Reply:
done